### DISCUSSION TITLE
Welcome to Bforartists Discussions!

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#2085] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Draise14** [comment]
I'm Draise, a technical 3D artists or an all round generalist. I've been using Bforartists a few years now, but used Blender a few years before then. I was using Softimage many years, and trueSpace before then - loving the UI's and UX of those software. I tried Maya and a tiny bit of Max, but opted for Bforartists and Blender for the opensource community. I've started doing 3D since I was 14 years old, now my principal business is working for clients and doing short animations and game asset packs, occasionally managing a small team of artists while at it. Every now and then I get a personal piece out - but all in all, I spend hundreds of hours in Bforartists, testing it and using it strenuously. Call me the beta tester and Bforartists Evangelist! Apparently in other Discord servers I've become a little infamous for touting this fork a lot - but most who try it agree it is very nice, having made some good friends along the way. 

On top of that, I currently mod and run the official Discord server, also the official Twitter and Facebook pages - though the socials need more publicity and activity to really grow as much as I have managed to grow activity in Discord. 

I currently sponsor a developer for Bforartists also. I often report feature requests and every now and then do an easy bug fix. I also report feedback from users from the socials or the Discord server if they aren't resolved through Blender bug reporting or other UX workflows. 

Here is some of my favorite animations that I've done and my art vs the artists: 
![ArtvsArtists_02](https://user-images.githubusercontent.com/35357409/101641955-6858e780-3a00-11eb-8980-366974c07c2e.jpg)
